app/models/automation/condition.rb

```diff
@@ -165,4 +169,23 @@ def pdf_match?(object, value)
       pdf_string.match?(value)
     end
   end
+
+  class APIConnectionCondition < Automation::Condition
```

Comment:
tuto rails convencie su nedava za sebou viac velkych pismen lebo to rozbije vselico ine. dajme `APIConnectionCondition`